### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#3559]
Our Uni won't allow us to use Galaxy until certain accessibility features are improved
GTN-specific notes are:

"My conclusion is that the training material is more accessible than the Galaxy platform itself but there are some areas that keyboard and screen reader users cannot accesses.  These are collapsed text boxes containing tips or answers to questions that cannot be expanded"

**Comment by hexylena** [comment]
ok! I was wondering about that specifically since I realised the other day we don't annoucne when boxes are expanded/collapsed. I'll look into that

**Comment by hexylena** [comment]
Todos:

- [ ] Replace boxes with custom html elements (e.g. `<tip title="Blah">...</tip>`)
  - This removes need for blockquotes completely (at cost of updating every tutorial)
  - Makes validation **much** easier
  - Removes a11y issue of incorrect heading levels (e.g. tip in h1 goes h1→h3)
  - Needs a translation layer in jekyll.
- [ ] Make boxes announce that they're opening/closing content below them.
  - can we use summary/details here? are those accessible by default?
  - either way need to check and announce with aria properties

**Comment by hexylena** [comment]
fixed in #3633
